app/scripts/filters/util.js

```diff
@@ -514,4 +514,22 @@ angular.module('openshiftConsole')
     return function(feature) {
       return _.get(Constants, ['ENABLE_TECH_PREVIEW_FEATURE', feature], false);
     };
-  });
+  })
+  .filter('mobileClientIconClass', function() {
+    return function(mobileClientResource) {
+      var clientIcon = _.get(mobileClientResource, 'metadata.labels.icon');
```

Comment:
This really feels like it should be an annotation, not a label. Other resources use annotations for icons.

Reply:
will add as annotation to the CRD